conans/test/functional/subsystems_build_test.py

```diff
@@ -1,13 +1,18 @@
 import platform
 
 import pytest
+import textwrap
 
+from conans.test.assets.sources import gen_function_cpp, gen_function_h
 from conans.test.utils.tools import TestClient
 
 
 @pytest.mark.skipif(platform.system() != "Windows", reason="Tests Windows Subsystems")
 class TestSubsystemsBuild:
 
+    def setup(self):
```

Comment:
Try to stop using old-style setup methods. Use ``pytest`` fixtures instead.
Typically it is not necessary to use classes if they don't provide a useful classification of tests.

Reply:
changed to `pytest.fixture`